client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -184,19 +188,22 @@ const CheckoutThankYou = React.createClass( {
 
 	render() {
 		let purchases = null,
+			failedPurchases = null,
 			wasJetpackPlanPurchased = false,
 			wasDotcomPlanPurchased = false;
 		if ( this.isDataLoaded() && ! this.isGenericReceipt() ) {
 			purchases = getPurchases( this.props );
+			failedPurchases = getFailedPurchases( this.props );
 			wasJetpackPlanPurchased = purchases.some( isJetpackPlan );
 			wasDotcomPlanPurchased = purchases.some( isDotComPlan );
 		}
 
-		const userCreatedMoment = moment( this.props.userDate );
-		const isNewUser = userCreatedMoment.isAfter( moment().subtract( 2, 'hours' ) );
+		const userCreatedMoment = moment( this.props.userDate ),
+			isNewUser = userCreatedMoment.isAfter( moment().subtract( 2, 'hours' ) ),
+			goBackText = this.props.selectedSite ? this.translate( 'Back to my site' ) : this.translate( 'Register domain' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **36** times:
`translate( 'Register Domain' )` *ES Score: 16.93*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/add-thank-you-route-for-domain-only)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).